src/main/scala/net/bzzt/reproduciblebuilds/Certification.scala

```diff
@@ -58,7 +57,7 @@ case class Certification(
   )
 
   def asPropertyString: String = {
-    val packageName = groupId + ":" + artifactId
+    groupId + ":" + artifactId
```

Comment:
I guess this was dead code entirely?

Reply:
Looks like it, shall I remove the line entirely?

Reply:
Done and rebased/pushed.